web/db/weapons.ts

```diff
@@ -145,6 +145,7 @@ add(registerMessage({ defaultMessage: "Whiteblind" }), Guyun, [FatuiKnife, Treas
 // Mist Veiled Elixir
 add(registerMessage({ defaultMessage: "Blackcliff Pole" }), MistVeiledElixir, [MistGrass, FatuiInsignia]);
 add(registerMessage({ defaultMessage: "Blackcliff Slasher" }), MistVeiledElixir, [MistGrass, FatuiInsignia]);
+add(registerMessage({ defaultMessage: "Calamity Queller" }), MistVeiledElixir, [MistGrass, WhopperflowerNectar], true);
```

Comment:
This item leads to [this wiki page](https://genshin-impact.fandom.com/wiki/Calamity_Queller) which doesn't exist yet. It should be solved when the wiki itself is updated for this weapon after some time. Just leaving this here as a remark, since it's not actionable.